### ISSUE TITLE
Commend Count not Updating

### ISSUE THREAD

**Issue by ezrinkenji711** [ValveSoftware/Dota2-Gameplay#12143]
Just a minor bug but I think it really needs attention for a fix. My commend counts are not updating. Let me know if you need further details.

**Comment by pablopeguero** [comment]
yep. I noticed this too. Its interesting how there seem to be a lot of problems popping up. Its been several days now that monkey kings aghs just doesnt work. 4200 gold item that literally doesnt serve its function.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/11278

**Comment by paololinaja89** [comment]
i'm having a same issue my commend count is not updating for a long time.

**Comment by kutaykartoz** [comment]
Can this be fixed already??? and fix it retroactively if possible..
This issue is here for more than a month. Valve even implemented new custom profiles and new behavior system, but didn't care about fixing one of its essential ingredient.
